src/models/users.js

```diff
@@ -1,6 +1,46 @@
 const mongoose = require("mongoose");
 Schema = mongoose.Schema;
 
+const addressSchema = new Schema({
+    address_id: {
+        type: Schema.Types.ObjectId,
+        required: true,
+        unique: true,
+        default: mongoose.Types.ObjectId // Generate a new ObjectId by default
+    },
+    user_id: {
+        type: Schema.Types.ObjectId,
+        required: true,
+        ref: 'user', //refrencing the address to the user. 
```

Comment:
The ref for the user in the address schema should be 'User' instead of 'user' since it should match the model name.